hwdb.d/60-keyboard.hwdb

```diff
@@ -1438,6 +1438,11 @@ evdev:input:b0003v046DpC309*
  KEYBOARD_KEY_c01b6=images                              # My Pictures (F11)
  KEYBOARD_KEY_c01b7=audio                               # My Music (F12)
 
+# Logitech MX Keys for Mac
+evdev:input:b0003v046Dp4092*
+ KEYBOARD_KEY_70035=102nd
+ KEYBOARD_KEY_70064=grave
```

Comment:
Could you add short comments for each key like other entries?

Reply:
Sure thing, so basically just
```
 KEYBOARD_KEY_70035=102nd                               # <
 KEYBOARD_KEY_70064=grave                               # ^
```
or what kind of comment do you have in mind?

Reply:
Not bad, but only `<` in the comment looks slightly cryptic. How about something like `# '<' key` ?

Reply:
Makes sense, done!

Thanks for the quick reply